share/static/html/spawn.html

```diff
@@ -67,6 +69,20 @@ <h1 style="font-size: 55px">everware</h1>
 require(["jquery"], function ($) {
   // add bootstrap form-control class to inputs
   $("#spawn_form").find("input, select, textarea, button").addClass("form-control");
+  function showError(message) {
+    $("#error_text").text(message);
```

Comment:
Should this be `$("#error_text").text("Error: " + message);` to stay consistent with `Error: {{error_message}}`